src/common/ipfs.ts

```diff
@@ -1,6 +1,8 @@
-import FormData from "form-data";
 import { MetadataURIOrObject } from "../core/types";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const FormData = require("form-data");
```

Comment:
can we do this with a switch on

```ts
if(!globalThis.FormData){
  globalThis.FormData = require("form-data");
}
```

Reply:
reason: this will otherwise break some web builds (because "form-data" lib includes node internal packages)

Reply:
@atbe this is really the only real blocker from my side

Reply:
Hmm, so should I be replacing this line with: ??

```js
if(!globalThis.FormData){
  globalThis.FormData = require("form-data");
}
```
I'm not familiar with the `globalThis` keyword (googling it now)

Reply:
Oh I see, so if this is true, we're running in nodejs because otherwise `FormData` would be defined on the `globalThis` object

Reply:
Alright I just tested this on console, all is well 👍

Reply:
yep exactly